relay-general/src/processor/selector.pest

```diff
@@ -10,7 +10,7 @@ Or = _{ "||" | "|" }
 Not = _{ "~" | "!" }
 EscapedQuote = @{ "'" }
 
-UnquotedKey = @{ (ASCII_ALPHANUMERIC | "-" | "_") + }
+UnquotedKey = @{ !ASCII_DIGIT ~ (ASCII_ALPHANUMERIC | "-" | "_") + }
```

Comment:
Shouldn't this also prevent leading special characters?

```suggestion
UnquotedKey = @{ ASCII_ALPHA ~ (ASCII_ALPHANUMERIC | "-" | "_") * }
```

Reply:
I thought `_foo` would be fine. like identifiers work in prog langs

Reply:
Understood, as you prefer then. Pest would give you `XID_START` and `XID_CONTINUE`, although those don't contain `"-"` and identifiers don't start with a minus.

Reply:
I'm not sure, yeah. I think I would want it because of header names? I'll think about it a couple days.

Reply:
let's just have them quote header names, it's fine